src/main/resources/application.conf

```diff
@@ -1,12 +1,88 @@
-akka {
-  event-handlers = ["akka.event.slf4j.Slf4jEventHandler"]
-  loggers = ["akka.event.slf4j.Slf4jLogger"]
-  loglevel = "DEBUG"
-  actor {
-    debug {
-      receive = on
-      event-stream = on
-      fsm = on
+webservice.port = 8000
+webservice.interface = 0.0.0.0
+instance.name = "reference"
+
+swagger {
+  apiDocs = "api-docs"
+  apiVersion = "0.1"
+  baseUrl = "/"
+  contact = "dsde@broadinstitute.org"
+  description = "One workflow service to rule them all..."
+  info = "Cromwell"
+  license = "BSD"
+  licenseUrl = "http://opensource.org/licenses/BSD-3-Clause"
+  swaggerVersion = "1.2"
+  termsOfServiceUrl = "http://www.github.com/broadinstitute/cromwell"
+}
+
+spray.can {
+  server {
+    request-timeout = 40s
+  }
+  client {
+    request-timeout = 40s
+    connecting-timeout = 40s
+  }
+}
+
+
+backend {
+  backend = "JES" // Either "JES" or "local", case insensitive
```

Comment:
Yes - I'm making some changes related to the whole config structure which will take that into account